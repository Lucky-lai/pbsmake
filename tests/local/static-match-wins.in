
static: dynamic1
	echo static

dynamic%: static${pm_target_match}
	echo dynamic: ${pm_target_match}

static1: dynamic2
	echo static1

static2:
	echo static2

dynamic2:
	echo dynamic2
