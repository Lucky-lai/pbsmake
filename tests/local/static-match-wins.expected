dynamic2
static1
dynamic: 1
static
